src/main/scala/org/hammerlab/guacamole/assembly/AssemblyArgs.scala

```diff
@@ -1,9 +1,8 @@
 package org.hammerlab.guacamole.assembly
 
-import org.bdgenomics.utils.cli.Args4jBase
 import org.kohsuke.args4j.{Option => Args4jOption}
 
-trait AssemblyArgs extends Args4jBase {
```

Comment:
while moving `Args` traits around I decided the right place to mix in `Args4jBase` is at lower levels of the inheritance tree, e.g. in the actual `SparkCommand`s' Arguments that use them.

I've ended up with a few traits that serve as interfaces to "args" traits that implement fields as `@Argument`s, which also helped clarify that we don't need to be specifying every thing up the tree as a concrete "args" object.